requirements.txt

```diff
@@ -0,0 +1 @@
+argparse
```

Comment:
```suggestion
argparse
boto3
```
For those installing packages from local to run

Reply:
No you don't want to do this because it causes error on the lambda as it always has boto3. The tox file notes that it needs boto3.

Reply:
Running `make unittest` or `tox` in the root works with a new python install -> I know, I did it on my accenture mac.